source/guide_hedgedoc.rst

```diff
@@ -0,0 +1,395 @@
+  [...]
+  100%[======================================================>] 50,784,713  16.8MB/s   in 2.9s
+  [isabell@stardust ~]$ tar --extract --gzip --file=hedgedoc-$VERSION.tar.gz
+  [isabell@stardust ~]$ rm --verbose hedgedoc-$VERSION.tar.gz
+  removed hedgedoc-x.x.x.tar.gz
+  [isabell@stardust ~]$
+
+Setup
+-----
+
+Then install the dependencies. This step may take a few minutes.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ cd hedgedoc
+  [isabell@stardust hedgedoc]$ bin/setup
+  Copying config files...
+  Installing packages...
+  yarn install v1.22.5
+  [1/5] Validating package.json...
+  [2/5] Resolving packages...
+  [3/5] Fetching packages...
+  info There appears to be trouble with your network connection. Retrying...
+  info fsevents@1.2.13: The platform "linux" is incompatible with this module.
+  info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
+  info fsevents@2.1.3: The platform "linux" is incompatible with this module.
+  info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
+  [4/5] Linking dependencies...
+  [5/5] Building fresh packages...
+  Done in 253.95s.
+  yarn install v1.22.5
+  [1/5] Validating package.json...
+  [2/5] Resolving packages...
+  success Already up-to-date.
+  Done in 1.15s.
+
+
+  Edit the following config file to setup HedgeDoc server and client.
+  Read more info at https://github.com/hedgedoc/hedgedoc#configuration-files
+
+  * config.json           -- HedgeDoc config
+  * .sequelizerc          -- db config
+  [isabell@stardust hedgedoc]$
+
+Database
+--------
+
+Create a Database.
+
+.. code-block:: console
+
+  [isabell@stardust hedgedoc]$ mysql --verbose --execute="CREATE DATABASE ${USER}_hedgedoc"
+  --------------
+  CREATE DATABASE isabell_hedgedoc
+  --------------
+
+  [isabell@stardust hedgedoc]$
+
+Configuration
+=============
+
+First we remove the config file that was generated by the setup because it might be very confusing. It was a copy of ``config.json.example`` if you like to have a look at it.
+
+.. code-block:: console
+
+
+  [isabell@stardust hedgedoc]$ rm --verbose config.json
+  removed ‘config.json’
+  [isabell@stardust hedgedoc]$
+
+.. note::
+
+  You can set here some options in json format you can't yet set with environment variables. For other configuration options, check the `configuration documentation`_.
+
+Create ``~/hedgedoc/config.json`` with the following content:
+
+.. code-block:: json
+
+  {}
+
+Edit the file ``~/hedgedoc/.sequelizerc`` with the following content:
+
+.. code-block:: js
+  :emphasize-lines: 7
+
+  var path = require('path');
+
+  module.exports = {
+      'config':          path.resolve('config.json'),
+      'migrations-path': path.resolve('lib', 'migrations'),
+      'models-path':     path.resolve('lib', 'models'),
+      'url':             process.env.CMD_DB_URL
```

Comment:
What do you think about specifying the actual mysql connection string here, as suggested in [the official docs](https://docs.hedgedoc.org/setup/manual-setup/#instructions)? That way users don't need to use it twice for the `manage_users` command later.

Reply:
Okay, **I change it back for the time**.

This was something like a preparation for the upcoming 2.0 release where all settings can be set as environment variables in `.env` files in the app root directory. As mentioned in the issue #876 (latest comment) I was aware of that problem. The development for 2.0 will take some weeks or months. I'm sure the guide will then need some rework. They are also working on their CLI tool(s), but I think it will still be very limited with the 2.0 release.